tolstoy/src/schema.rs

```diff
@@ -29,6 +37,12 @@ pub fn ensure_current_version(tx: &mut rusqlite::Transaction) -> Result<()> {
         tx.execute(statement, &[])?;
     }
 
+    // Initial partition information is what we'd see at bootstrap, and is used during first sync.
+    // For now, only information about the user partition is tracked.
+    // We rely on the transactor to advance the tx partition.
+    // Support for migrating the db partition forward (that is, bootstrap schema) is TBD.
+    tx.execute("INSERT OR IGNORE INTO tolstoy_parts VALUES (?, ?, ?)", &[&PARTITION_USER, &USER0, &USER0])?;
```

Comment:
We also have https://github.com/mozilla/mentat/blob/5fe4f12d32379415e36eb80877cbeec6fd443ffa/db/src/bootstrap.rs#L88.